server/routes/friendLists/friendLists.route.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const passport = require('passport');
+const mongoose = require('mongoose');
+const router = express.Router({ mergeParams: true });
+const FriendList = require('../../models/friendList');
+
+// All calls respond with the updated array of lists
+function getUserLists(userId) {
+  return new Promise((resolve, reject) => {
+    FriendList.find({ userId }, (err, docs) => {
+      if (err) reject(err);
+      FriendList.populate(docs, {
+        path: 'friendIds',
+        select: ['name', 'active'],
+      })
+        .then((result) => resolve(result))
+        .catch((err) => reject(err));
+    });
+  });
+}
```

Comment:
I'd add this to a file called `util.js` and nest it under `friendLists` directory

Reply:
Makes sense! I'll do that

Reply:
Actually, now looking at all the routes, following that logic the polls endpoint should be in /users too, because you're getting either the user's polls or the user's friends' polls, so it uses the user id.

Reply:
Honestly if I was doing this myself I'd just have the routes without any sub-children, because we are not using any shared params and all routes (except auth) should be private and access the user from the JWT token. What do you think?

Reply:
If by children, you mean sub-directories, I agree. I think it would be much cleaner to have each route have a directory of its own with its accompanying `util.js` file. We could add this as an icebox issue